### PR TITLE
Created header

### PR DESCRIPTION
**Pull request by PinkMoon25**
Created the portfolio page by using Figma Template 2, 
Added linters for HTML and CSS
created header and the headline section using flexbox, 
exported background image from Figma and added it to the headline section, 
used social media icons and hover effect on link elements, and used Figma inspect to get the right font size and colors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <p class="description">I can help you build a product, feature or website.
+      Look through some of my work and experience! If you like what you see and have a project you need coded,
+      don't hestiate to contact me.</p>
+    <nav class="social-media" rel="noopener">
+      <a href="https://github.com/PinkMoon25" target="_blank" rel="noopener"><i class="fa-brands fa-github"></i></a>
+      <a href="https://www.linkedin.com/in/tushar-singh-6b063a14b/" target="_blank" rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a>
+      <a href="https://twitter.com/TusharS90674484"><i class="fa-brands fa-twitter" rel="noopener"></i></a>
+      <a href="#blank"><i class="fa-solid fa-m" rel="noopener"></i></a>
+      <a href="#peace"><i class="fa-solid fa-hand-peace"></i></a>
+    </nav>
```
Comment (Munsa1) [inline index.html:28]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag). This is the correct HTML structure, I would advise you to follow it

- [ ] Also the requirements state that you should use flexbox to align these items, kindly do so.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @PinkMoon25 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
To be honest your project is one of the best I have seen so far just a few semantic issues and you are done.


## Highlights
:heavy_check_mark: Linters
:heavy_check_mark: Descriptive readme file
:heavy_check_mark: Correct gitflow
:heavy_check_mark: Beautiful UI design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/5ff773556d.js" crossorigin="anonymous"></script>
+  <title>Portfolio Page</title>
+</head>
+<body>
+  <header class="header">
+    <h1>Tushar Singh</h1>
```
[the remark this answers is not in the document]
Comment (PinkMoon25) [reply index.html]: > Hi @PinkMoon25,
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! To be honest your project is one of the best I have seen so far just a few semantic issues and you are done.

Thank You @Munsa1 I'm glad it means a lot, I put some late-night effort into this, and as requested I have made the semantic changes.

File: index.html (hunk 1 above)
Comment (PinkMoon25) [reply index.html]: Hi, @Munsa1 I have used the ul tag to group the links inside list items, and also used flexbox to align the items.

**Review by Kandy-Peter — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @PinkMoon25 ,  

Your project is complete! Great job for nailing it right on the second attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- A good descriptive PR  ✔️
- Nice Ghtub Flow



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀

**Pull request merged**